### PR TITLE
dizay updated its website name

### PR DESCRIPTION
**Pull request by emirkabal**

### CODE REVIEW

File: websites/Y/YabancıDiziizlesene/presence.ts [hunk 1]
```diff
@@ -5,34 +5,19 @@ const presence = new Presence({
...
-
-const startTimestamp = Math.floor(Date.now() / 1000);
-
+  }),
+  startTimestamp = Math.floor(Date.now() / 1000);
+  
 let video: HTMLVideoElement;
 
-presence.on("iFrameData", async (msg) => {
+presence.on("iFrameData", async (msg: any) => {
```
Comment (ririxidev) [inline websites/Y/YabancıDiziizlesene/presence.ts:13]: no any type

File: websites/Y/YabancıDiziizlesene/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "Kabal",
+    "id": "269581360316940299"
+  },
+  "service": "YabancıDiziizlesene",
+  "description": {
+    "tr": "YabancıDiziizlesene, 1080p dizi izleme imkanını sunan, online yabancı dizi ve film izleme sitesidir.",
+    "en": "YabancıDiziizlesene is an online series and movie viewing site that offers the possibility of watching 1080p."
+  },
+  "url": "yabancidiziizlesene.com",
+  "version": "1.1.5",
```
Comment (ririxidev) [inline websites/Y/YabancıDiziizlesene/dist/metadata.json:12]: 1.1.6

File: websites/Y/YabancıDiziizlesene/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "logo": "https://i.imgur.com/fTVFRFj.png",
```
Comment (ririxidev) [inline websites/Y/YabancıDiziizlesene/dist/metadata.json:13]: can't you get better logo quality?

**Review by ririxidev — CHANGES_REQUESTED** [review]
you should look at these things:

File: websites/Y/YabancıDiziizlesene/presence.ts [hunk 4]
```diff
@@ -5,34 +5,19 @@ const presence = new Presence({
...
-
-const startTimestamp = Math.floor(Date.now() / 1000);
-
+  }),
+  startTimestamp = Math.floor(Date.now() / 1000);
+  
 let video: HTMLVideoElement;
 
-presence.on("iFrameData", async (msg) => {
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
```
Comment (ririxidev) [inline websites/Y/YabancıDiziizlesene/presence.ts:13]: You can't assign this type to this message lol, you are not sending video element, you are sending duration, currenttime and paused state, so do custom interface for it... and assign it.

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/Y/YabancıDiziizlesene/iframe.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+var iframe = new iFrame();
+
```
Comment (Bas950) [inline websites/Y/YabancıDiziizlesene/iframe.ts:2]: no var

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — CHANGES_REQUESTED** [review]
image
image
image

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**